content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,54 @@ title: Custom Metrics
 kind: documentation
 ---
 
-If your Lambda functions are sending trace or log data to Datadog, and the data you want to query is captured in an existing log or trace, you can [generate custom metrics from logs and traces](#creating-custom-metrics-from-logs-or-traces) without re-deploying or making any changes to your application code. If you want to send new data not captured in an existing log or trace, you can [send custom metrics using the Datadog Lambda Library](#sending-custom-metrics-from-the-datadog-lambda-library).
+## Overview
 
-## Creating custom metrics from logs or traces
-
-With log-based metrics, you can record a count of logs that match a query or summarize a numeric value contained in a log, such as request duration. Log-based metrics are a cost-efficient way to summarize log data from the entire ingest stream. Learn more about creating log-based metrics [here][1]. 
-
-You can also generate metrics from 100% of ingested spans, regardless of whether they are indexed by a retention filter. Learn more about creating span-based metrics [here][2].
-
-## Sending custom metrics from the Datadog Lambda library
-
-{{< img src="serverless/serverless_custom_metrics.png" alt="Collecting Custom Metrics from AWS Lambda" >}}
+There are a few different ways to submit custom metrics to Datadog from a Lambda function. 
 
-Install the Datadog Lambda Library to collect and send custom metrics. Metrics sent from the Datadog Lambda Library are automatically aggregated into [distributions][3], so you can graph the `avg`, `sum`, `max`, `min`, and `count`. You can also calculate aggregations over a set of tags for the 50th, 75th, 95th, and 99th percentile values on the [Distribution Metrics][3] page.
+- **Creating custom metrics from logs or traces**: If your Lambda functions are already sending trace or log data to Datadog, and the data you want to query is captured in an existing log or trace, you can [generate custom metrics from logs and traces](#creating-custom-metrics-from-logs-or-traces) without re-deploying or making any changes to your application code.
+- **Submitting custom metrics using the Datadog Lambda Extension**: If you want to submit custom metrics directly from your Lambda function, Datadog recommends using the [Datadog Lambda Extension](#with-the-datadog-lambda-extension). Check whether the Datadog Lambda Extension is supported for the runtime of your Lambda function [here][1].  
```

Comment:
```suggestion
- **Submitting custom metrics using the Datadog Lambda Extension**: If you want to submit custom metrics directly from your Lambda function, Datadog recommends using the [Datadog Lambda Extension](#with-the-datadog-lambda-extension). [Check whether the Datadog Lambda Extension is supported][1] in your Lambda function runtime.  
```

Reply:
cc @tianchu I'm told to avoid using "here" for links for a few reasons:
- it's wordy, and you could just tie the link to the action
- it's not as good for screen readers